app/src/main/res/layout/layer_dialog.xml

```diff
@@ -0,0 +1,47 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Give a screenshot in the PR of what this looks like

Reply:
![5bf6ece40e4a0341caf5dae541f49af5](https://user-images.githubusercontent.com/33967726/192074979-fa3e2134-6f39-42a3-9e8e-87eee01baa22.png)

Reply:
Add padding so the buttons aren't at the bottom
Also, default in Android the OK button is on the right

Reply:
done:
![5bf6ece40e4a0341caf5dae541f49af5](https://user-images.githubusercontent.com/33967726/192076506-9cf1fab9-e52b-411b-844e-65ba1bab92df.png)

Reply:
done
![e0028598988e8a3158bc9613982c1023](https://user-images.githubusercontent.com/33967726/192076522-157ad109-b278-4cfa-b66c-71e8e295b018.png)